.travis.yml

```diff
@@ -6,7 +6,7 @@ after_success:
   - npm run codecov
 before_deploy:
   - cd docs
-  - npm ci
+  - npm install
```

Comment:
Since the `docs/` directory *doesn't* currently have a lock file, `npm ci` won't work – should consider adding a lock file for the docs though